maths/greatest_common_divisor_alt.py

```diff
@@ -0,0 +1,31 @@
+"""
+Greater Common Divisor.
+https://en.wikipedia.org/wiki/Euclidean_algorithm#Implementations
+"""
+
+
+def gcd(a, b):
+    """Calculate Greater Common Divisor (GCD)."""
+    if a = 0:
```

Comment:
Hey @Prathamesh404! You need to replace this with `a == 0`

Reply:
It would also be great if you added some doctests. You can refer to this link:
https://docs.python.org/2/library/doctest.html

Reply:
Should I create a different commit altogether?
Or should I amend this?

Reply:
Amend this one.  Please make sure that Travis tests are passing.  See CONTRIBUTING.md for details.